### ISSUE TITLE
"Priming" Learning rate 3e-4 not working for layers greater than 16

### ISSUE THREAD

**Issue by afiaka87** [lucidrains/deep-daze#39]
We discussed this elsewhere, but just to be rigorous -

As it stands, I think priming only works on about 16-20 layers. Otherwise, the loss gets stuck in the 0.08 range. I found it's able to escape this 0.08 value by lowering the learning rate. 

Now what would really be nice is if we found good rates for certain layer counts. In the meantime, I just made it tweakable from the Imagine interface and the CLI. Here's the code - 

https://github.com/lucidrains/deep-daze/pull/38

**Comment by afiaka87** [comment]
> Off topic: It's tough to figure out what SIREN + CLIP will "latch onto", but faces is _definitely_ one of those things. It's very good at taking the existing face of an image and converting it to the face of your description, so long as the face is popular enough at least. Here's worf from star trek on a photo of Mac's mom from the show It's Always Sunny in Philadelphia
> 
> Input:
> ![macsmom](https://user-images.githubusercontent.com/3994972/106562449-ecef5000-64ef-11eb-8eca-8dcd044f329f.jpeg)
> 
> SIREN representation:
> ![macsmom](https://user-images.githubusercontent.com/3994972/106562573-17410d80-64f0-11eb-8c5f-0019d952efff.png)
> 
> After 100 iterations of training on "worf":
> ![worf_after](https://user-images.githubusercontent.com/3994972/106562824-83237600-64f0-11eb-8602-cf1b99fa0729.png)
> 
> Early output(phrase: "worf"):

**Comment by afiaka87** [comment]
@lucidrains thanks!
